package.json

```diff
@@ -45,7 +45,7 @@
     "chai": "^4.3.6",
     "klona": "^2.0.5",
     "mocha": "^10.0.0",
-    "vc-test-suite-implementations": "github:w3c/vc-test-suite-implementations"
+    "vc-test-suite-implementations": "../vc-test-suite-implementations"
```

Comment:
this is odd, why are you installing implementations from a local dir?

Reply:
Sorry...yeah. That wasn't supposed to sneak in there. My bad.